style.css

```diff
@@ -0,0 +1,6 @@
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table; }
+.clearfix:after {
+  clear: both; }
+/*# sourceMappingURL=style.css.map */
```

Comment:
Hmm i'm not sure about how to get it to add the blank link ill have to look into it. Even github view complains about the lack of blank line so i'm sure there must be a way to get it to add it.